### DISCUSSION TITLE
I want to contribute my bank's import config, how do I provide two configs for the same bank?

### DISCUSSION THREAD

**Discussion by Dave4234** [firefly-iii/firefly-iii#5041] [Firefly III]
I'm wondering the correct way to provide multiple import configs for one bank to the repository here: https://github.com/firefly-iii/import-configurations

My bank provides a different format of CSV for my main account vs my credit card account. The instructions say they should all be called "default.json". Should I name one "default.json" and use a different name for the second format? I randomly clicked through and couldn't find any for other banks that weren't named "default.json".

**Comment by JC5** [comment]
Yes, that's OK. Name it `cc.json`. If you want, create a file called `readme.md` in the same directory and explain the difference. 

An example can be found [here](https://github.com/firefly-iii/import-configurations/tree/main/de/dkb)

**Reply by Dave4234** [reply]
Awesome, thanks! The example is very helpful, I'll follow their lead.
